sycl/doc/extensions/proposed/sycl_ext_oneapi_graph_fusion.asciidoc

```diff
@@ -162,33 +202,53 @@ this buffer will inherit this property (unless overridden).
 `sycl::aligned_alloc_device()`, `sycl::malloc_shared()`, or
 `sycl::aligned_alloc_shared()` to apply the property to an USM pointer.
 
-```c++ 
-sycl::ext::oneapi::experimental::property::promote_local 
+```c++
+namespace sycl::ext::oneapi::experimental::property{
+
+  template<sycl::memory_scope Scope>
+  struct access_scope {};
+
+  inline constexpr auto access_scope_local = 
+                                    access_scope<memory_scope_work_group>;
+
+  inline constexpr auto access_scope_private = 
+                                    access_scope<memory_scope_work_item>;
```

Comment:
I don't feel strongly enough about this to block the PR, but I think using "local" and "private" in the names here might be a mistake.  We're `access_scope_private` isn't as clear as `access_scope<memory_scope_work_item>` -- without reading the specification, it's not clear what the access is private _to_.  Somebody could easily misunderstand this as "access is private to the kernel", for example.

`access_scope_work_group` and `access_scope_work_item` might be a good balance between clarity and verbosity.

Reply:
Renamed the shortcuts as you proposed.